### PR TITLE
Bug 1181535 - Deactivate the tests on Aries until they get fixed

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9627258788825bddf1b018c9e3c84f4e6adb947e

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini [hunk 1]
```diff
@@ -76,10 +76,12 @@ skip-if = device == "desktop"
 dsds = true
 # Bug 1168072 - Intermittent failures in Jenkins in test_dialer_dsds_make_phone_call_with_default_sim.py
 stable = false
-skip-if = device == "desktop"
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini]: You're removing this skip-if for desktop, why?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini [hunk 2]
```diff
@@ -76,10 +76,12 @@ skip-if = device == "desktop"
...
+# Bug 1182525 - The aries device doesn't support DSDS
+skip-if = device == "aries"
 
 [test_dialer_dsds_receive_call_on_both_sims.py]
 dsds = true
 # Bug 1168071 - Intermittent failures in Jenkins in test_dialer_dsds_receive_call_on_both_sims.py
 stable = false
-skip-if = device == "desktop"
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini]: You're removing this skip-if for desktop, why?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini (hunk 1 above)
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini]: I started to clean up, like explained in bug 1189747. I'm fine to not do it now and wait for bug 1189747.

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/fmradio/manifest.ini [hunk 3]
```diff
@@ -5,18 +5,23 @@ antenna = true
 external = false
 
 [test_fmradio_add_remove_from_favorites.py]
-skip-if = device == "desktop"
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/fmradio/manifest.ini]: You're removing this skip-if for desktop, why?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/fmradio/manifest.ini [hunk 4]
```diff
@@ -5,18 +5,23 @@ antenna = true
...
+# Bug 1182111 - [Aries] No FM radio test can run, because FM radio doesn't work on Z3C
+skip-if = device == "aries"
 
 [test_fmradio_find_stations.py]
 disabled = Bug 1000863 - Investigate test_fmradio_find_stations.py failure
-skip-if = device == "desktop"
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/fmradio/manifest.ini]: You're removing this skip-if for desktop, why?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/fmradio/manifest.ini [hunk 5]
```diff
@@ -5,18 +5,23 @@ antenna = true
...
+# Bug 1182111 - [Aries] No FM radio test can run, because FM radio doesn't work on Z3C
+skip-if = device == "aries"
 
 [test_fmradio_frequency_dialer.py]
-skip-if = device == "desktop"
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/fmradio/manifest.ini]: You're removing this skip-if for desktop, why?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/fmradio/manifest.ini [hunk 6]
```diff
@@ -5,18 +5,23 @@ antenna = true
...
+# Bug 1182111 - [Aries] No FM radio test can run, because FM radio doesn't work on Z3C
+skip-if = device == "aries"
 
 [test_fmradio_turn_on_off.py]
-skip-if = device == "desktop"
+# Bug 1182111 - [Aries] No FM radio test can run, because FM radio doesn't work on Z3C
+skip-if = device == "aries"
 
 [test_fmradio_airplane_mode.py]
-skip-if = device == "desktop"
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/fmradio/manifest.ini]: You're removing this skip-if for desktop, why?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/gallery/manifest.ini [hunk 7]
```diff
@@ -8,40 +8,62 @@ sdcard = true
...
+skip-if = device == "aries"
 
 [test_gallery_share_picture.py]
+# Bug 1182086 - [Aries] navigator.getDeviceStorage('sdCard') doesn't return the same sdCard we have in the Flame
+skip-if = device == "aries"
 
 [test_gallery_switch_to_camera.py]
 # Bug 1111637 - Create a new test case for the switching to camera from gallery and back
 camera = true
-skip-if = device == "desktop"
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/gallery/manifest.ini]: You're removing this skip-if for desktop, why?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/manifest.ini [hunk 8]
```diff
@@ -13,9 +13,10 @@ smoketest = true
 camera = true
 
 [test_lockscreen_unlock_to_camera_with_passcode.py]
-skip-if = device == "desktop"
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/manifest.ini]: You're removing this skip-if for desktop, why?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/manifest.ini [hunk 9]
```diff
@@ -26,7 +26,8 @@ skip-if = device == "desktop"
 [test_sms_with_attachments.py]
 camera = true
 smoketest = true
-skip-if = device == "desktop"
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/manifest.ini]: You're removing this skip-if for desktop, why?

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/manifest.ini [hunk 10]
```diff
@@ -30,7 +30,8 @@ online = true
 smoketest = true
 
 [test_settings_cell_data.py]
-skip-if = device == "desktop"
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/manifest.ini]: You're removing this skip-if for desktop, why?

**Comment by JohanLorenzo** [comment]
Mass closing PRs. Gaia is not a supported project anymore.

**Pull request closed without merging**